docker-compose.yml

```
@@ -0,0 +1,57 @@
+      - node_modules:/app/node_modules
+    ports:
+      - 3000:3000
+    networks:
+      - todo-net
+
+networks:
+  todo-net:
+volumes:
+  postgres_data:  # データベースの永続化用ボリューム
+  maven_cache:    # Mavenキャッシュ用ボリューム
+  node_modules:   # Node.jsの依存関係用ボリューム
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)